kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -178,6 +185,13 @@ private[v1] class SessionsResource extends ApiRequestContext {
     }
   }
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON
+    )),
+    description = "Create an operation with GET_TYPE_INFO type"
+  )
   @POST
   @Path("{sessionHandle}/operations/typeinfo")
```

Comment:
From the previous discussion, could you update to camel case style?
```scala
@Path("{sessionHandle}/operations/typeInfo")
```

Reply:
I have already merged this .. can you send a followup ? thank you !

Reply:
OK